posts/Yoann/2016-09-30-introducing-hibernate-search-sort-dsl.adoc

```diff
@@ -0,0 +1,88 @@
+= Introducing Hibernate Search Sort DSL
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search" ]
+:awestruct-layout: blog-post
+---
+With Elasticsearch support coming as a technological preview in Hibernate Search 5.6, you would think we're leaving out other features. Well, think again! Enters the Sort DSL, which will work with Elasticsearch of course, but also with the good ol' Lucene backend.
+
+The point here is to provide an API to build sort descriptions easily, without knowing everything about Hibernate Search features added on top of Lucene, such as `DistanceSortField`. And while we're at it, we're making it a modern, fluid API.
+
+== Overview
+
+The API works pretty much like the query builder. The query builder is the starting point, actually: a new `sort()` method has been added to the `QueryBuilder` interface. Here's what it looks like:
+
+[source, JAVA]
+----
+QueryBuilder builder = fullTextSession.getSearchFactory()
+  .buildQueryBuilder().forEntity(Book.class).get();
+Query luceneQuery = /* ... */;
+FullTextQuery query = fullTextSession.createFullTextQuery( luceneQuery, Book.class );
+Sort sort = builder
+  .sort()
+    .byField("author").desc() // Descending order
+    .andByField("title") // Default order (ascending)
+  .createSort();
+query.setSort(sort);
+List results = query.list();
+----
+
+Of course, other options are available. For instance, you may stabilize your sort by calling `andByIndexOrder` after your field sorts:
+
+[source, JAVA]
+----
+  .sort()
+    .byField("title")
+    .andByIndexOrder()
+  .createSort()
+----
+
+That way, if there are two books with the same title in your index, they will always keep the same relative order from one query to another.
+
+The score sort is also available with `byScore()`. Obviously, there's one key difference with that one: the sort is descending by default. Getting the least relevant entities first wouldn't be very useful.
```

Comment:
Instead of point out the obvious I would explain how you can change the order of the score..
Something like:
`If you need the least relevant results, fear not, we got you covered,  with the byScore().desc()` method.`